src/services/functionAppService.ts

```diff
@@ -173,25 +161,58 @@ export class FunctionAppService extends BaseService {
       }
     };
 
-    await this.sendFile(requestOptions, functionZipFile);
-    this.serverless.cli.log("-> Function package uploaded successfully");
+    await this.sendFile(requestOptions, this.functionZipFile);
+    
+    this.log("-> Function package uploaded successfully");
     const serverlessFunctions = this.serverless.service.getAllFunctions();
     const deployedFunctions = await this.listFunctions(functionApp);
 
-    this.serverless.cli.log("Deployed serverless functions:")
+    this.log("Deployed serverless functions:")
     deployedFunctions.forEach((functionConfig) => {
       // List functions that are part of the serverless yaml config
       if (serverlessFunctions.includes(functionConfig.name)) {
         const httpConfig = this.getFunctionHttpTriggerConfig(functionApp, functionConfig);
 
         if (httpConfig) {
           const method = httpConfig.methods[0].toUpperCase();
-          this.serverless.cli.log(`-> ${functionConfig.name}: ${method} ${httpConfig.url}`);
+          this.log(`-> ${functionConfig.name}: ${method} ${httpConfig.url}`);
         }
       }
     });
   }
 
+  /**
+   * Uploads artifact file to blob storage container 
+   */
+  private async uploadFunctionAppToBlobStorage() {
```

Comment:
I would call this `uploadZipArtifactToBlobStorage` or `uploadFunctionArtifactToBlobStorage`